dotcom-rendering/cypress/support/commands.js

```diff
@@ -25,3 +25,21 @@ import 'cypress-wait-until';
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('hydrate', () => {
+	return cy
+		.get('gu-island')
+		.each((el) => {
+			cy.wrap(el)
+				.log(`Scrolling to ${el.attr('name')}`)
+				.scrollIntoView({ duration: 100, timeout: 10000 })
+				.should('have.attr', 'data-gu-ready', 'true', {
+					timeout: 30000,
+				});
+		})
+		.then(() => {
+			cy.scrollTo('top');
+			// eslint-disable-next-line cypress/no-unnecessary-waiting
+			cy.wait(5000);
```

Comment:
I'm guessing you've already thought this through but is there something in the dom that we can check for here rather than having this wait?

Reply:
Although we assert on all islands being 'ready' this is additional safety to ensure that the page has stopped shifting and fully loaded post hydration. 

I thought about waiting for x amount of idle time but that feels error prone in itself 

Without adding more state to islands rendering I can't think of a signal we could use to achieve this 🤔